qutebrowser/config/configdata.yml

```diff
@@ -1030,6 +1030,42 @@ input.spatial_navigation:
     Right key, heuristics determine whether there is an element he might be
     trying to reach towards the right and which element he probably wants.
 
+input.blur_on_load.enabled:
+  default: true
+  type: Bool
+  desc: >-
+    Clear focus on page during page load.
+
+    This prevents the page from stealing focus on page load, similar to how
+    js vim plugins preserve their own focus.
```

Comment:
I don't get "how js vim plugins preserve their own focus" - after reading it a few times, I think you mean e.g. Vimperator? I think it'd be better to just drop that sentence.